charts/vault/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v2
+name: vault
+description: Vault Helm Chart
+type: application
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.0.0
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application.
+appVersion: latest
```

Comment:
Would it make sense not to update it to the M4D version but rather match it to the appVersion of the subchart?

Reply:
yes, it makes sense. I will change that `appVersion: 1.6.2` which is the vault appVersion for Vault Version 0.9.1, leaving `version: 0.0.0.`